tests/neptune/new/attributes/atoms/test_artifact.py

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
Add it as a comment to Jira or in additional PR and throw away. It's not really reliable and someone might don't remember to update it properly.